DEVELOPER_GUIDE.md

```diff
@@ -37,7 +37,7 @@ pytest tests
 ### Serve the Examples
 
 ```bash
-panel serve examples/app_*.py --dev
+panel serve $(find examples -name "*.py") --dev
```

Comment:
I renamed the dashboards from `app.py` to more specific names to enable serving them together.